Mathlib/NumberTheory/ModularForms/EisensteinSeries/UniformConvergence.lean

```diff
@@ -152,8 +152,7 @@ lemma rpow_bound {k : ℝ} (hk : 0 ≤ k) (z : ℍ) (x : Fin 2 → ℤ) (hx : x
       simp
       rw [mem_box_ne_zero_iff_ne_zero n x (by rw [Int.mem_box])] at hx
       norm_cast at *
-  rw [h11]
-  simp only [Nat.cast_max, ofReal_int_cast, map_mul, abs_ofReal, ge_iff_le]
+  simp only [h11, Nat.cast_max, ofReal_int_cast, map_mul, abs_ofReal, ge_iff_le]
```

Comment:
You still have a lot of spurious lemmas in your simp calls
```suggestion
  simp only [h11, Nat.cast_max, ofReal_int_cast, map_mul, abs_ofReal]
```

Reply:
hmm weird, I'll re-squeeze them! thanks